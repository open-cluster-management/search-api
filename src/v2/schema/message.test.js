import supertest from "supertest";
import server, { GRAPHQL_PATH } from "../index";


describe("Message Resolver", () => {
  test("Correctly Resolves Message Query", () =>
    new Promise((done) => {
      supertest(server)
        .post(GRAPHQL_PATH)
        .send({
          query: `
        {
          messages {
            id
            kind
            description
          }
        }
      `,
        })
        .end((err, res) => {
          expect(JSON.parse(res.text)).toMatchSnapshot();
          done();
        });
    }));
});
