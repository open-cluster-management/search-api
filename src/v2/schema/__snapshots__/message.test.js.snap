// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message Resolver Correctly Resolves Message Query 1`] = `
Object {
  "data": Object {
    "messages": Array [
      Object {
        "description": "Search is disabled on some of your managed clusters.",
        "id": "S20",
        "kind": "information",
      },
    ],
  },
}
`;
